stat/sampleuv/example_rate_test.go

```diff
@@ -24,26 +24,17 @@ func ExampleMetropolisHastings_samplingRate() {
 	// Successive samples are correlated with one another through the
 	// Markov Chain defined by the proposal distribution. To get less
 	// correlated samples, one may use a sampling rate, in which only
-	// one sample from every few is accepted from the chain. This can
-	// be accomplished through a for loop.
+	// one sample from every few is accepted from the chain.
```

Comment:
"few" seems unusually imprecise here. I'm not sure if that's a problem, but it seems strange.

Reply:
Rewrote.